applications/aero_acoustic/co_rotating_vortex_pair/application.h

```diff
@@ -793,27 +796,28 @@ class Application : public ApplicationBase<dim, Number>
   {
   }
 
-private:
-  void
-  set_single_field_solvers(std::string input_file, MPI_Comm const & comm) final
-  {
-    this->acoustic =
-      std::make_shared<AcousticsAeroAcoustic::Application<dim, Number>>(input_file, comm);
-    this->fluid = std::make_shared<FluidAeroAcoustic::Application<dim, Number>>(input_file, comm);
-  }
-
   void
   set_field_functions() final
   {
     this->field_functions->source_term_blend_in =
       std::make_shared<BlendInFunction<dim>>(this->acoustic->get_parameters().start_time,
-                                             0.1 * (this->acoustic->get_parameters().end_time -
-                                                    this->acoustic->get_parameters().start_time));
+                                             this->acoustic->get_parameters().start_time +
+                                               0.1 * (this->acoustic->get_parameters().end_time -
+                                                      this->acoustic->get_parameters().start_time));
 
     this->field_functions->analytical_aero_acoustic_source_term =
       std::make_shared<AnalyticalSourceTerm<dim>>(source_term_with_convection, intensity, r_0, r_c);
   }
 
+private:
+  void
+  set_single_field_solvers(std::string input_file, MPI_Comm const & comm) final
```

Comment:
This name should be changed. It is not the solvers that are set here, but the applications.

-> `set_single_field_applications()`

Reply:
I think other modules in ExaDG for coupled problems do not have such a function. I think the convention in ExaDG is currently that the constructor of a concrete `Application` has to create instances of concrete single-field applications, see https://github.com/exadg/exadg/blob/024b1224410ddd01a12ed924e107ee84838e1d14/applications/incompressible_flow_with_transport/differentially_heated_cavity/application.h#L504-L512. 

However, I also agree if you prefer such a private member function defined in the base class `ApplicationBase`. In this case, I nevertheless believe we should try to keep the convention that this is done in the constructor of the `Application` class as I have explained in other comments.